### ISSUE TITLE
AttributeError: module 'tensorflow' has no attribute 'get_variable'

### ISSUE THREAD

**Issue by hamediut** [genforce/interfacegan#100]
![InterFaceGAN_error](https://user-images.githubusercontent.com/47457060/176236352-e1f95701-7d10-48be-ad71-523c119f5e07.JPG)
Hello,
I am trying to implement your code but I keep getting this error despite changing all 'tf.' functions to 'tf.compat.v1.'
I solved couple of no module error but I cannot find this one anywhere in the .py files. Is it something happens upon loading the model?
Thank you in advance for your help :)

**Comment by Evelyn92** [comment]
Yes I met this issue before. It turned out that I did not load the correct pretrained model (.pth file). I put the .pth file in the pretrain folder as below and the issue was solved. Hope this could help.


![image](https://user-images.githubusercontent.com/52853830/236824557-d079382b-1990-40eb-ab01-b471f5efddc0.png)

**Comment by elizurkelner** [comment]
@Evelyn92 
Could you kindly either upload the "stylegan_celebahq.pth" file, or provide me with a link to download it?
